vscode-extension/src/prompts/agentPrompt.ts

```diff
@@ -0,0 +1,102 @@
+import * as fs from 'fs';
```

Comment:
Rename this from agentPrompt.ts to workflowInitPrompt.ts, as this is all very specific to workflow initialization and the file name is too generic.